evm-bridging/src/BridgedTopShotMoments.sol

```diff
@@ -73,31 +117,33 @@ contract BridgedTopShotMoments is
         return _customSymbol;
     }
 
-    function safeMint(address to, uint256 tokenId, string memory uri) public onlyOwner {
```

Comment:
Removed `safeMint` - minting will only be possible as part of:
1. Wrapper functionality
2. Bridge fulfillment (when contract is onboarded to upgraded bridge)